lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -794,6 +794,40 @@
 
     const BestFitSupportedLocales = LookupSupportedLocales;
 
+    /**
+     * Applies digit options used for number formatting onto the given intlObj
+     *
+     * This function is used by both NumberFormat and PluralRules, despite being defined
+     * as a NumberFormat abstract operation
+     *
+     * ECMA 402: #sec-setnfdigitoptions
+     *
+     * @param {Object} intlObj
+     * @param {Object} options
+     * @param {Number} mnfdDefault
+     * @param {Number} mxfdDefault
+     */
```

Comment:
descriptions for these params might help because I don't know immediately what mxfd means